src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  max-width: 200px;
+}
+
+.product-picture {
+  height: 134px;
+  width: 160px;
+  margin-top: 16px;
+}
+
+.product-name {
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.card__reviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  gap: 17px;
```

Comment:
redundant as you use space-between and card isn't responsive. Check for other cases
```suggestion
```